typescript.js

```diff
@@ -47,6 +49,7 @@ module.exports = {
     },
   },
   rules: {
+    'no-return-await': 'off',
```

Comment:
since an async function returns a promise anyway, why disable this rule. I don't understand the stacktrace issue

Reply:
Just found a blog to illustrate the issue ;)
https://www.even.li/posts/2020-06-04-async-stack-traces/

Reply:
It's pretty common to not return await, it's a good rule